Better error message when MSBuild fails with error

When MSBuild exists with error during initial compilation I see the following error:

```
\15.0\Bin\Microsoft.Common.CurrentVersion.targets(1657,5): error MSB4036: The "GetReferenceNearestTargetFrameworkTask" task was not found.
```

I fixed this issue by following fix 2 on this stack overflow question:

https://stackoverflow.com/questions/47797510/the-getreferencenearesttargetframeworktask-task-was-not-found

But I only found this solution after @Mobrockers pointed out to me that I could reproduce this error using MSBuild.exe

My suggestion: Add this error:

> Something went wrong during initial build. You can reproduce this error your self using MSBuild: `"C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild" MyProject\MyProject.sln"

--- a/src/Stryker.Core/Stryker.Core/Initialisation/InitialBuildProcess.cs
+++ b/src/Stryker.Core/Stryker.Core/Initialisation/InitialBuildProcess.cs
@@ -1,4 +1,4 @@
-﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging;
 using Stryker.Core.Exceptions;
 using Stryker.Core.Logging;
 using Stryker.Core.Testing;
@@ -41,18 +41,23 @@ namespace Stryker.Core.Initialisation
 
                 // Build project with MSBuild.exe
                 result = _processExecutor.Start(solutionDir, msbuildPath, $"\"{solutionPath}\"");
+                CheckBuildResult(result, msbuildPath, $"\"{solutionPath}\"");
             }
             else
             {
                 // Build with dotnet build
                 result = _processExecutor.Start(projectPath, "dotnet", $"build \"{Path.GetFileName(projectPath)}\"");
+                CheckBuildResult(result, "dotnet build", $"\"{Path.GetFileName(projectPath)}\"");
             }
+        }
 
+        private void CheckBuildResult(ProcessResult result, string buildCommand, string buildPath)
+        {
             _logger.LogDebug("Initial build output {0}", result.Output);
             if (result.ExitCode != 0)
             {
                 // Initial build failed
-                throw new StrykerInputException(result.Output, "Initial build of targeted project failed. Please make targeted project buildable. See above message for build output.");
+                throw new StrykerInputException(result.Output, $"Initial build of targeted project failed. Please make targeted project buildable. You can reproduce this error your self using \"{buildCommand} {buildPath}\"");
             }
             _logger.LogDebug("Initial build successful");
         }

--- a/src/Stryker.Core/Stryker.Core/Initialisation/InitialBuildProcess.cs
+++ b/src/Stryker.Core/Stryker.Core/Initialisation/InitialBuildProcess.cs
@@ -57,7 +57,7 @@ namespace Stryker.Core.Initialisation
             if (result.ExitCode != 0)
             {
                 // Initial build failed
-                throw new StrykerInputException(result.Output, $"Initial build of targeted project failed. Please make targeted project buildable. You can reproduce this error your self using \"{buildCommand} {buildPath}\"");
+                throw new StrykerInputException(result.Output, $"Initial build of targeted project failed. Please make sure the targeted project is buildable. You can reproduce this error yourself using: \"{buildCommand} {buildPath}\"");
             }
             _logger.LogDebug("Initial build successful");
         }

